### ISSUE TITLE
Unable to use different profiles for different games with the same executable name.

### ISSUE THREAD

**Issue by EmpiresBane** [PeterTh/gedosato#276]
This specific problem cropped up because of Tales from the Borderlands. The executables for both Borderlands 1 and Tales from the Borderlands are both named "Borderlands", preventing you from creating separate profiles for them.

**Comment by DAOWAce** [comment]
Same thing with a lot of other games that title their exes simple as 'game.exe'.

Welcome to what we've had to deal with even all the way back to the days of nHancer for NVIDIA drivers.

The only real way to get unique entries is to include a part of the folder path leading up to the exe, as is done in video driver profiles.  Maybe someone can add support for that to GeDoSaTo. (boy I wish I learned to code all those years ago)

Until then.. guess you gotta just rename configs depending on the game you want to play.

**Comment by SwinglesMan** [comment]
Try doing the launchers instead. I've never had this problem because I don't have any games with the same exe name :P

**Comment by PeterTh** [comment]
Yeah, I expected this to be a problem at some point. I guess I'll have to do the folder path thing when I get the time.

**Comment by SwinglesMan** [comment]
As long as it's not copy and paste the whole directory into the ini. Hope you can do that internally without changing the way we input the process names.

**Comment by PeterTh** [comment]
No, if I implement it at all it will be "one directory up" (which should be the same for everyone).

**Comment by PeterTh** [comment]
(closed by accident)
